packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -34,10 +34,17 @@ void describe('getting started happy path', async () => {
     // start a local npm proxy and publish the current codebase to the proxy
     await execa('npm', ['run', 'clean:npm-proxy'], { stdio: 'inherit' });
     await execa('npm', ['run', 'vend'], { stdio: 'inherit' });
+    /**
+     * delete .bin folder in the repo because
+     * 1) e2e tests don't use them
+     * 2) execa would use them if it can not find the binary in the test project
+     */
+    await execaCommand(`rm -rf ${process.cwd()}/node_modules/.bin`);
   });
 
   after(async () => {
     // stop the npm proxy
+    await execa('npm', ['run', 'setup:local'], { stdio: 'inherit' }); // add tsx back since we removed all the binaries
```

Comment:
What's the problem we are trying to solve here?

I'm also a little concerned about using `setup:local` in e2e tests. The intent of that script is to setup a local dev environment and coupling that to running e2e tests could lead to undesirable or unexpected behavior in the tests as that script evolves

Reply:
`./node_modules/.bin` was removed, but we need `tsx` to run `npm run stop:npm-proxy`.

I did `npm install tsx`, but it'd update package.json and package-lock.json with a newer version, which 1) engineer may accidentally include in a commit (I did), 2) have to run `npm run setup:local` again to install the binaries after the test.

Reply:
Could we just run `npm install` instead?